packages/dds/tree/api-report/tree.beta.api.md

```diff
@@ -274,11 +274,18 @@ interface ReadonlyMapInlined<K, T extends Unenforced<ImplicitAllowedTypes>> {
     values(): IterableIterator<TreeNodeFromImplicitAllowedTypesUnsafe<T>>;
 }
 
-// @public
+// @public @deprecated
 export type RestrictiveReadonlyRecord<K extends symbol | string, T> = {
     readonly [P in symbol | string]: P extends K ? T : never;
 };
 
+// @public
```

Comment:
do you know why @system isn't showing up here?

Reply:
this only shows specific tags that api-extractor knows about (ex: does not show remarks, link, param apparently system)